### ISSUE TITLE
Can/should we add a stylelint rule, forcing grid children to set a min-width

### ISSUE THREAD

**Issue by peeke** [kaliberjs/build#209]
Is it possible to add a stylelint rule, that forces grid children to define `min-width`? 

> By default, a grid item cannot be smaller than the size of its content, since grid items have an initial size of min-width: auto and min-height: auto.

This can (and will, at some point) lead to grid blowouts. Setting the `min-width: 0` effectively resets this minimum size to a less problematic value. The issue can also be fixed by using `minmax(0, 1fr)` instead of `1fr` in your column definitions, but this would in my eyes be less develoepr friendly.

https://css-tricks.com/preventing-a-grid-blowout/

**Comment by EECOLOR** [comment]
I don't think we have that kind of requirement yet, but I'm pretty sure the [`checkRuleRelation`](https://github.com/kaliberjs/build/blob/2466b985539f480e190ebf79991cc1c39314b376/library/stylelint-plugins/machinery/relations.js#L5) function can handle it

You can check [parent-child-policy](https://github.com/kaliberjs/build/blob/2466b985539f480e190ebf79991cc1c39314b376/library/stylelint-plugins/rules/parent-child-policy/index.js#L188) for some examples the other way around.
